makefiles/Makefile.docs-node

```diff
@@ -23,10 +23,8 @@ help: ## Show this help message
 	@echo 'Variables'
 	@printf "  \033[36m%-18s\033[0m %s\n" 'ARGS' 'Arguments to pass to mut-publish'
 
-publish: remote-includes ## Builds this branch's publishable HTML and other artifacts under build/public
-	if [ ${GIT_BRANCH} = master ]; then rm -rf build/master build/public; fi
-	giza make publish
-	if [ ${GIT_BRANCH} = master ]; then mut-redirects config/redirects -o build/public/.htaccess; fi
+next-gen-publish: next-gen-html
```

Comment:
If you add a comment with two leading hashes, e.g. `next-gen-publish: next-gen-html ## ...`, then this rule will be listed in `make help`